docs/tutorials/TracingCode.md

```diff
@@ -87,26 +92,26 @@ Now let’s set the breakpoint. First, double-click the item to reach the corres
 
 Recall from the User Guide that the `edit` command has the format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​` For this tutorial we will be issuing the command `edit 1 n/Alice Yeoh`.
 
-<div markdown="span" class="alert alert-primary">
+<box type="tip">
 
-:bulb: **Tip:** Over the course of the debugging session, you will encounter every major component in the application. Try to keep track of what happens inside the component and where the execution transfers to another component.
-</div>
+**Tip:** Over the course of the debugging session, you will encounter every major component in the application. Try to keep track of what happens inside the component and where the execution transfers to another component.
+</box>
 
 1. To start the debugging session, simply `Run` \> `Debug Main`
 
-1. When the GUI appears, enter `edit 1 n/Alice Yeoh` into the command box and press `Enter`.
+2. When the GUI appears, enter `edit 1 n/Alice Yeoh` into the command box and press `Enter`.
 
-1. The Debugger tool window should show up and show something like this:<br>
+3. The Debugger tool window should show up and show something like this:<br>
    ![DebuggerStep1](../images/tracing/DebuggerStep1.png)
 
-1. Use the _Show execution point_ feature to jump to the line of code that we stopped at:<br>
+4. Use the _Show execution point_ feature to jump to the line of code that we stopped at:<br>
    ![ShowExecutionPoint](../images/tracing/ShowExecutionPoint.png)<br>
    `CommandResult commandResult = logic.execute(commandText);` is the line that you end up at (i.e., the place where we put the breakpoint).
 
-1. We are interested in the `logic.execute(commandText)` portion of that line so let’s _Step in_ into that method call:<br>
+5. We are interested in the `logic.execute(commandText)` portion of that line so let’s _Step in_ into that method call:<br>
     ![StepInto](../images/tracing/StepInto.png)
 
-1. We end up in `LogicManager#execute()` (not `Logic#execute` -- but this is expected because we know the `execute()` method in the `Logic` interface is actually implemented by the `LogicManager` class). Let’s take a look at the body of the method. Given below is the same code, with additional explanatory comments.
+6. We end up in `LogicManager#execute()` (not `Logic#execute` -- but this is expected because we know the `execute()` method in the `Logic` interface is actually implemented by the `LogicManager` class). Let’s take a look at the body of the method. Given below is the same code, with additional explanatory comments.
```

Comment:
```suggestion
1. We end up in `LogicManager#execute()` (not `Logic#execute` -- but this is expected because we know the `execute()` method in the `Logic` interface is actually implemented by the `LogicManager` class). Let’s take a look at the body of the method. Given below is the same code, with additional explanatory comments.
```